spec/models/image_spec.rb

```diff
@@ -1,5 +1,5 @@
 require 'rails_helper'
 
 RSpec.describe Image, :type => :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it{should belong_to :product}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.